<launch>
  <arg name="namespace" default="docker" />

  <arg name="verbose" default="true" />
  <arg name="out_debug_img" default="true" />
  <arg name="out_debug_segimage" default="true" />
  <arg name="confidence_threshold" default="0.5" />
  <arg name="input_image" default="input" />
  <arg name="compressed" default="false" />

  <include file="$(find detic_ros)/launch/decompress.launch" if="$(arg compressed)">
    <arg name="in" value="$(arg input_image)"/>
  </include>

  <group ns='$(arg namespace)'>
    <node name="detic_segmentor"
          pkg="detic_ros" type="node.py"
          output="screen" >
    <remap from="~input_image" to="$(arg input_image)"/>
    <param name="verbose" value="$(arg verbose)"/>
    <param name="out_debug_img" value="$(arg out_debug_img)"/>
    <param name="out_debug_segimage" value="$(arg out_debug_segimage)"/>
    <param name="confidence_threshold" value="$(arg confidence_threshold)"/>
    </node>
  </group>
</launch>
